src/Components/Authorization/LoginForm.tsx

```diff
@@ -95,7 +95,7 @@ export const LoginForm: FormComponentType = props => {
                 <Text color={Colors.grayDark}>Remember me</Text>
               </Checkbox>
               <ForgotPasswordLink
-                handleClick={props.handleChangeMode("reset_password")}
+                onClick={() => props.handleChangeMode("reset_password")}
```

Comment:
we should also rename `handleChangeMode` to something clearer like `presentModal`